Invalid type for license
Prevent variables relevant only for a standalone python-pkg
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -8,7 +8,6 @@
 name = "spglib"
 requires-python = ">=3.9"
 description = "This is the spglib module."
-license = "BSD-3-Clause"
 license-files = ["COPYING"]
 readme = "python/README.rst"
 dynamic = ["version"]
@@ -89,16 +88,6 @@
 metadata.version.provider = "scikit_build_core.metadata.setuptools_scm"
 sdist.include = ["python/spglib/_version.py"]
 
-[tool.scikit-build.cmake.define]
-SPGLIB_WITH_Python = "ON"
-SPGLIB_WITH_TESTS = "OFF"
-SPGLIB_USE_OMP = "OFF"
-
-[[tool.scikit-build.overrides]]
-if.env.CIBUILDWHEEL = true
-# Make sure cibuildwheel builds with a bundled spglib
-cmake.define.CMAKE_DISABLE_FIND_PACKAGE_Spglib = "ON"
-
 [tool.setuptools_scm]
 write_to = "python/spglib/_version.py"
 
